### DISCUSSION TITLE
dexcom bridge

### DISCUSSION THREAD

**Discussion by djhurlburt** [NightscoutFoundation/xDrip#2633] [Q&A]
I am wondering how to get the nightscout running on the Google Cloud (as established here: https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud.html) can leverage the Dexcom Bridge and ingest dexcom straight from the share capability.

**Comment by Navid200** [comment]
After you complete installation, Nightscout will be the same as Nightscout on any other server.
The only possible difference is explained here:  https://navid200.github.io/xDrip/docs/Nightscout/Auth_Default_Roles.html

There is nothing wrong with having that particular question posted here.
But, we have set up a platform allowing the installation of the official Nightscout in it.
Questions about how to use Nightscout can be better answered in other channels.  One example is the CGM in the cloud support group on facebook:  https://www.facebook.com/groups/cgminthecloud
